src/xcode/ENA/ENA/Source/Coordinator.swift

```diff
@@ -0,0 +1,282 @@
+				risk: state.risk,
+				exposureSubmissionService: self.exposureSubmissionService
+			)
+		}
+
+		self.homeController = homeController
+
+		UIView.transition(with: rootViewController.view, duration: CATransaction.animationDuration(), options: [.transitionCrossDissolve], animations: {
+			self.rootViewController.setViewControllers([homeController], animated: false)
+		})
+
+		#if !RELEASE
+		enableDeveloperMenuIfAllowed(in: homeController)
+		#endif
+	}
+
+	func showOnboarding() {
+		rootViewController.navigationBar.prefersLargeTitles = false
+		rootViewController.setViewControllers(
+			[
+				AppStoryboard.onboarding.initiateInitial { [unowned self] coder in
+					OnboardingInfoViewController(
+						coder: coder,
+						pageType: .togetherAgainstCoronaPage,
+						exposureManager: self.exposureManager,
+						store: self.store
+					)
+				}
+			],
+			animated: false
+		)
+	}
+
+	func updateState(detectionMode: DetectionMode, exposureManagerState: ExposureManagerState, risk: Risk?) {
+		homeController?.updateState(detectionMode: detectionMode, exposureManagerState: exposureManagerState, risk: risk)
+	}
+
+	#if !RELEASE
+	private var developerMenu: DMDeveloperMenu?
+	private func enableDeveloperMenuIfAllowed(in controller: UIViewController) {
+		developerMenu = DMDeveloperMenu(
+			presentingViewController: controller,
+			client: client,
+			store: store,
+			exposureManager: exposureManager
+		)
+		developerMenu?.enableIfAllowed()
+	}
+	#endif
+
+	private func setExposureManagerEnabled(_ enabled: Bool, then completion: @escaping (ExposureNotificationError?) -> Void) {
+		if enabled {
+			exposureManager.enable(completion: completion)
+		} else {
+			exposureManager.disable(completion: completion)
+		}
+	}
+}
+
+extension Coordinator: HomeViewControllerDelegate {
+	func showRiskLegend() {
+		rootViewController.present(
+			AppStoryboard.riskLegend.initiateInitial(),
+			animated: true,
+			completion: nil
+		)
+	}
+
+	func showExposureNotificationSetting(enState: ENStateHandler.State) {
+		let storyboard = AppStoryboard.exposureNotificationSetting.instance
+		let vc = storyboard.instantiateViewController(identifier: "ExposureNotificationSettingViewController") { coder in
+			ExposureNotificationSettingViewController(
+					coder: coder,
+					initialEnState: enState,
+					store: self.store,
+					delegate: self
+			)
+		}
+		addToUpdatingSetIfNeeded(vc)
+		rootViewController.pushViewController(vc, animated: true)
+	}
+
+	func showExposureDetection(state: HomeInteractor.State, isRequestRiskRunning: Bool) {
+		let state = ExposureDetectionViewController.State(
+			exposureManagerState: state.exposureManagerState,
+			detectionMode: state.detectionMode,
+			isLoading: isRequestRiskRunning,
+			risk: state.risk
+		)
+		let vc = AppStoryboard.exposureDetection.initiateInitial { coder in
+			ExposureDetectionViewController(
+				coder: coder,
+				state: state,
+				delegate: self
+			)
+		}
+//		delegate?.addToUpdatingSetIfNeeded(vc)
```

Comment:
dead code

Reply:
This was copied over. I left it as it was because there might be a reason someone left that commented-out. It's not a dead code, it's commented-out code. Dead code needs to be executable to be dead code and commented-out code isn't executable. :)

Reply:
This line should be removed now as it doesn't exist in develop anymore.